src/Options/Elements/Line.php

```diff
@@ -0,0 +1,326 @@
+<?php
+
+namespace Halfpastfour\PHPChartJS\Options\Elements;
+
+
```

Comment:
Please bring this back to one empty line.

Reply:
done